packages/tasks/src/model-libraries.ts

```diff
@@ -275,6 +275,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "mindspore",
 		repoUrl: "https://github.com/mindspore-ai/mindspore",
 	},
+	"mars5-tts": {
```

Comment:
Let's get some repos to https://huggingface.co/models?other=mars5-tts before merging

Reply:
Opened a PR: https://huggingface.co/CAMB-AI/MARS5-TTS/discussions/9